.cirrus.yml

```diff
@@ -0,0 +1,31 @@
+task:
+  env:
+    CFLAGS: -I/usr/local/include
+    LDFLAGS: -L/usr/local/lib
+    ibmtpm_name: ibmtpm1119
```

Comment:
Any reason we can't use the same version of the simulator as the travis-ci config?

Reply:
That's changed to align with travis.